airflow/migrations/versions/a13f7613ad25_resource_based_permissions_for_default_.py

```diff
@@ -0,0 +1,172 @@
+    ],
+    ("RoleModelView", "can_edit"): [
+        (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_ROLE),
+    ],
+    ("RoleModelView", "can_add"): [
+        (permissions.ACTION_CAN_CREATE, permissions.RESOURCE_ROLE),
+    ],
+    ("RoleModelView", "can_copyrole"): [
+        (permissions.ACTION_CAN_CREATE, permissions.RESOURCE_ROLE),
+    ],
+    ("ViewMenuModelView", "can_list"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_VIEW_MENU),
+    ],
+    ("UserDBModelView", "can_add"): [
+        (permissions.ACTION_CAN_CREATE, permissions.RESOURCE_VIEW_MENU),
+    ],
+    ("UserDBModelView", "can_userinfo"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_MY_PROFILE),
+    ],
+    ("UserDBModelView", "can_download"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_VIEW_MENU),
+    ],
+    ("UserDBModelView", "can_show"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_VIEW_MENU),
+    ],
+    ("UserDBModelView", "can_list"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_VIEW_MENU),
+    ],
+    ("UserDBModelView", "can_edit"): [
+        (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_VIEW_MENU),
+    ],
+    ("UserDBModelView", "resetmypassword"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_MY_PASSWORD),
+    ],
+    ("UserDBModelView", "resetpasswords"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_PASSWORD),
+    ],
+    ("UserDBModelView", "userinfoedit"): [
+        (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_MY_PROFILE),
+    ],
+    ("UserDBModelView", "can_delete"): [
+        (permissions.ACTION_CAN_DELETE, permissions.RESOURCE_VIEW_MENU),
+    ],
+    ("UserInfoEditView", "can_this_form_get"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_MY_PROFILE),
+    ],
+    ("UserInfoEditView", "can_this_form_post"): [
+        (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_MY_PROFILE),
+    ],
+    ("UserStatsChartView", "can_chart"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_USER_STATS_CHART),
+    ],
+    ("UserLDAPModelView", "can_userinfo"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_MY_PROFILE),
+    ],
+    ("UserOAuthModelView", "can_userinfo"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_MY_PROFILE),
+    ],
+    ("UserOIDModelView", "can_userinfo"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_MY_PROFILE),
+    ],
+    ("UserRemoteUserModelView", "can_userinfo"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_MY_PROFILE),
+    ],
+    ("DagRunModelView", "can_clear"): [
+        (permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG),
+        (permissions.ACTION_CAN_DELETE, permissions.RESOURCE_TASK_INSTANCE),
+    ],
+}
+
+
+def remap_permissions():
+    """Apply Map Airflow view permissions."""
+    appbuilder = create_app(config={'FAB_UPDATE_PERMS': False}).appbuilder
```

Comment:
Should we also set `conf.getboolean('webserver', 'UPDATE_FAB_PERMS'):` somehow?

Reply:
@ashb What would be the purpose of that?

Reply:
I honestly couldn't tell you anymore.